runtime/src/parachains.rs

```diff
@@ -95,6 +98,22 @@ impl<T: Trait> ParachainRegistrar<T::AccountId> for Module<T> {
 }
 
 pub trait Trait: session::Trait {
+	/// The outer origin type.
+	type Origin: From<Origin> + From<system::RawOrigin<Self::AccountId>>;
+
+	/// The outer call dispatch type.
+	type Proposal: Parameter + Dispatchable<Origin=<Self as Trait>::Origin>;
```

Comment:
Why call it `Proposal`? I feel this is a bit too easily conflated with governance proposals. I'd imagine that parachains can dispatch those, but that's among other things.